style_editor.html

```diff
@@ -264,13 +270,61 @@
   border: 1px solid #ccc;
   border-top: none;
 }
+.structured-view {
+  position: sticky;
+  top: 0;
+  width: 100%;
+  height: 100vh;
+  overflow-y: scroll;
+}
 /* Structured View Layer buttons
 Add, Delete, Move Up, Move Down */
 .extra-buttons {
   margin: 3px;
   line-height: 1.0em;
   font-size: 16px;
 }
+/* Footer */
+.page-title {
```

Comment:
It's weird for this to be "page-title"
How about footer-title?

Reply:
Agreed. Remnant. Renamed.